src/main/resources/assets/toomanyorigins/lang/tr_tr.json

```diff
@@ -65,26 +78,26 @@
     "item.toomanyorigins.dragon_fireball": "Ejderha Ateştopu",
     "entity.toomanyorigins.fireball_area_effect_cloud": "Ateş Topu Alan Etkisi Bulutu",
     "death.attack.dragonMagic": "%1$s ejderha nefesinde kavruldu",
-    "death.attack.dragonMagic.player": "%1$s, %2$s dan kaçmaya çalışırken ejderha nefesinde kavrultuldu.",
-    "death.attack.dragonMagic.player.item": "%1$s, %3$s kullanan %2$sden kaçmaya çalışırken ejderha nefesinde kavrultuldu.",
-    "death.attack.indirectDragonMagic": "%1$s, %2$s tarafından ejderha nefesinde kavrultuldu.",
-    "death.attack.indirectDragonMagic.item": "%1$s, %3$s kullanan %2$s tarafından ejderha nefesinde kavrultuldu.",
+    "death.attack.dragonMagic.player": "%1$s, %2$s'den kaçmaya çalışırken ejderha nefesinde kavruldu.",
```

Comment:
it depends on the noun if it's den or dan, but the noun is a variable, the only solution to this problem is to cry, because glorius [Vowel Harmony](https://en.wikipedia.org/wiki/Vowel_harmony)

I just made it 'den because that's how they do it in the base game's translation